test/utils/FakeObjects.scala

```diff
@@ -204,10 +204,10 @@ trait FakeObjects {
   )
 
   val ownerAuthResultWithNoAuthorisations = OwnerAuthResult(
-    start = 1,
-    size = 10,
-    filterTotal = 10,
-    total = 10,
+    start = 0,
+    size = 0,
+    filterTotal = 0,
+    total = 0,
     authorisations = Seq.empty
```

Comment:
start and size = 0 are not realistic (allowed) values are they?

Reply:
can you give me an example of realistic values for when you are returning no authorisations?

Reply:
Yes, start = 1 and size = 15.

Reply:
how can start and size have those values when there are no auths? what do those numbers relate to ?

Reply:
Do you not know? You changed the values so I assumed you knew. start is the page number (min value 1), size is the page size (min value 10, default 15)

Reply:
i changed it to what I thought they should be. Are those values meant to be copied from the request? I don't see why we would set them to anything other than zero. Do those numbers not relate to the face that we are returning no results?

Reply:
I'm not sure I can explain any better than this:`start` is the page number (min value 1), `size` is the page size (min value 10, default 15)

Reply:
they are pagination parameters aren't they? which page you are on, and max results per page

Reply:
ive changed it to what you suggested (it still doesn't make sense to me but 🤷‍♂)

Reply:
@darrell-plant and @habeeb01 if you guys can have a quick call around this to clear up the confusion.

Reply:
When we make a request to a paginated endpoint , we supply the page number and page size. Page size is restricted to be between 10 and 100 (I think) and usually defaults to 15. page 0 doesn't exist :)
The response returns the page and size values that were supplied so that the client code can know where they are in the pagination and don't have to store previous values.

A response with no results will always be on page 1 and the requested page size would be 10 or 15 or ...